backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlSchemaFieldFactory.java

```diff
@@ -483,9 +495,15 @@ public GraphqlSchemaFieldFactory() {
 
   private static DataFetcher<?> queryFetcher(Schema schema) {
     return dataFetchingEnvironment -> {
+      // check for 'expanded' parameter
+      boolean enhance = false;
+      if (dataFetchingEnvironment.getArgument(GraphqlConstants.ENHANCE) != null
+          && dataFetchingEnvironment.getArgument(GraphqlConstants.ENHANCE).equals(true)) {
```

Comment:
prefer not to have a feature switch because we would like to get rid of that in the future again. Instead, accept that we need to make fixes to bootstrap so we don't create new legacy code.